src/Sarif.Multitool.FunctionalTests/TestData/MessagesShouldEndWithPeriod/Invalid.sarif

```diff
@@ -110,48 +110,6 @@
               ]
             }
           ],
-          "graphs": {
-            "g": {
-              "id": "g",
-              "description": {
-                "text": "graph description without period",
-                "richText": "_Rich_ graph description without period"
-              },
-              "nodes": [
-                {
-                  "id": "n1"
-                },
-                {
-                  "id": "n2"
-                }
-              ],
-              "edges": [
-                {
-                  "id": "e",
-                  "sourceNodeId": "n1",
-                  "targetNodeId": "n2"
-                }
-              ]
-            }
-          },
-          "graphTraversals": [
-            {
-              "graphId": "g",
-              "description": {
-                "text": "graphTraversal description without period",
-                "richText": "_Rich_ graphTraversal description without period"
-              },
-              "edgeTraversals": [
-                {
-                  "edgeId": "e",
-                  "message": {
-                    "text": "edgeTraversal message without period",
-                    "richText": "_Rich_ edgeTraversal message without period"
-                  }
-                }
-              ]
-            }
```

Comment:
ok, this is a controversial choice i made! in order for the associated test cases to pass afaik i had two options: 1. write a prerelease transformer, 2. remove these from our samples since we do not actually have a customer who uses this feature yet. 

If theres a third option which can help us preserve these cases, please let me know, i would be happy to implement :)

Reply:
the third option would be to preserve the test content but update it to the new standard. :)